src/pudl/transform/ferc1.py

```diff
@@ -2138,20 +2155,58 @@ def apply_xbrl_calculation_fixes(
                 ],
             },
             "balance_sheet_assets_ferc1": {
+                "current_and_accrued_assets": [
```

Comment:
These are the values that truly showed up in two different calculations, and so had to be duplicated.  However they **also** seem to have had the wrong sign associated with them in the calculation metadata.